paper_to_git/config/config.py

```diff
@@ -141,12 +141,12 @@ def ensure_directories_exist(self):
         if self.create_paths:
             for path_name, path in self.paths.items():
                 makedirs(path)
-            # Create a paper_git.cfg file if it already doesn't exist.
+            # Create a paper-git.cfg file if it already doesn't exist.
             lock_file = os.path.join(self.VAR_DIR, 'paper-git-cfg.lck')
-            paper_git_cfg = os.path.join(self.ETC_DIR, 'paper-git.cfg')
+            paper-git_cfg = os.path.join(self.ETC_DIR, 'paper-git.cfg')
```

Comment:
Same here, variables can't be named `paper-git_cfg`, it has to be `paper_git_cfg`. 

There can't be `-` in the variable names.

Python allows only a few characters in the variable names, `_` is one of them.